packages/apps/human-app/frontend/src/shared/helpers/regex.ts

```diff
@@ -3,9 +3,8 @@ export const password8Chars =
 export const passwordUppercase = /(?=.*[A-Z])/;
 export const passwordLowercase = /(?=.*[a-z])/;
 export const passwordNumeric = /(?=.*[0-9])/;
-export const passwordSpecialCharacter =
-  /(?=.*[$^*.[\]{}()?"!@#%&/\\,><':;|_~`=+-])/;
+export const passwordSpecialCharacter = /[!@#$%^&+=*]/; // allowed characters: @#$%^&+=!*]
 
-// this is sum of all above regular expressions
+// this is sum of all above regular expressions that matches with reputation oracle 'auth.dto.ts'
```

Comment:
We've already removed any restrictions except for min length from reputation oracle: https://github.com/humanprotocol/human-protocol/pull/2389/files

Reply:
done